docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.2'
+services:
+
+  fedora-25:
+    image: anton-povarov/pinba2:fedora-25
```

Comment:
This will allow you to build & push images with predefined names without providing them every time on command line, for example:

```
$ docker-compose build fedora-25
Building fedora-25
Step 1/13 : FROM        fedora:25
 ---> 9cffd21a45e3
Step 2/13 : MAINTAINER  Anton Povarov "anton.povarov@gmail.com"
 ---> Using cache
...
```
```
$ docker-compose push fedora-25
The push refers to repository [docker.io/anton-povarov/pinba2]
...
```